### PR TITLE
update v5.0 docs/nav

### PR DESCRIPTION
**Pull request by janjust**

### CODE REVIEW

File: doc/index.php [hunk 1]
```diff
@@ -12,11 +12,11 @@
 
 <p><ul>
 
-<li><a href="current/"><strong>v4.1 series</strong></a>.
-This documentation reflects the latest progression in the 4.1.x series.
+<li><a href="current/"><strong>v5.0 series</strong></a>.
```
Comment (jsquyres) [inline doc/index.php:15]: Shouldn't this go out to docs.open-mpi.org?

File: doc/index.php [hunk 2]
```diff
@@ -26,16 +26,20 @@
 
 </ul></p>
 
-<h2>Upcoming release series</h2>
+<!-- <h2>Upcoming release series</h2>
```
Comment (jsquyres) [inline doc/index.php:29]: Delete, don't comment out 😄 

Perhaps link to main and call it "v5.1".

**Review by jsquyres — CHANGES_REQUESTED** [review]

File: doc/index.php (hunk 1 above)
Comment (janjust) [reply doc/index.php]: you're right, fixed with forced update

File: doc/index.php (hunk 2 above)
Comment (janjust) [reply doc/index.php]: deleted

**Review by jsquyres — APPROVED** [review]

**Pull request merged**